cmd/ctl/pkg/convert/convert.go

```diff
@@ -159,7 +155,9 @@ func (o *Options) Complete() error {
 func (o *Options) Run() error {
 	builder := new(resource.Builder)
 
-	r := builder.Unstructured().LocalParam(true).ContinueOnError().
+	r := builder.
+		WithScheme(scheme, schema.GroupVersion{Group: cmapiv1alpha2.SchemeGroupVersion.Group, Version: runtime.APIVersionInternal}).
```

Comment:
Hm, come to think, this section of code only allows decoding things in the `cert-manager.io` group. Some of our resources are in the `acme.cert-manager.io` group too 👀  I think we can pass in some kind of 'group versioner' structure (which has always confused me a bit!) which allows us to accept multiple different groups.

We can follow up with this in another PR though as I think this does make good progress regardless 😄

Reply:
I'll open an issue for that then